setup.py

```diff
@@ -18,5 +18,5 @@ def read(fname):
       long_description_content_type='text/markdown',
       author='Andre Miras',
       url='https://github.com/AndreMiras/pycaw',
-      packages=['pycaw'],
+      packages=find_packages(),
```

Comment:
Don't we want to exclude tests and examples?
```suggestion
      packages=find_packages(exclude=("tests", "examples")),
```